### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by shraddha-amber**
## Home Page of Ecommerce - Fully Responsive

- The desktop hero section has static images, with text and CTA over it
image

- The mobile hero section has an image carousel 

- There are 3 featured products listed below the hero section
- Another CTA button for viewing all products
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <script defer src="/Shared/Navbar/navbar.js"></script>
   <script defer src="https://kit.fontawesome.com/f08ee55cc8.js" crossorigin="anonymous"></script>
+  <script defer src="/index.js"></script>
```
Comment (xsaurabhsuthar) [inline index.html]: You must use async in script tag.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+let slideIndex = 0;
+let imgSrc = [
+  "/Assets/HeroImages/evermore-carousel-img.png",
+  "/Assets/HeroImages/fearless-carousel-img.png",
+  "/Assets/HeroImages/red-carousel-img.png"
+];
+
```
Comment (xsaurabhsuthar) [inline index.js:7]: You can add some comments to describe the function.

**Review by xsaurabhsuthar — APPROVED** [review]
Looks perfect 🚀🚀.

File: index.html (hunk 1 above)
Comment (shraddha-amber) [reply index.html]: Okay, adding now

File: index.js (hunk 2 above)
Comment (shraddha-amber) [reply index.js:7]: Right, thanks for pointing that

**Review by ayush65 — APPROVED** [review]

**Pull request merged**